backend/de.metas.contracts/src/main/java/de/metas/contracts/invoicecandidate/FlatrateTerm_Handler.java

```diff
@@ -49,25 +52,26 @@ public class FlatrateTerm_Handler extends AbstractInvoiceCandidateHandler
 {
 	public final static String SYS_Config_AUTO_INVOICE = "de.metas.contracts.invoicecandidate.ALLOW_AUTO_INVOICE";
 
-	private final Map<String, ConditionTypeSpecificInvoiceCandidateHandler> conditionTypeSpecificInvoiceCandidateHandlers;
+	private final Multimap<String, ConditionTypeSpecificInvoiceCandidateHandler> conditionTypeSpecificInvoiceCandidateHandlers;
```

Comment:
it's a pitty that we don't have an enum for conditions types.... but c'est la vie, leave it like that